src/core/layout/qgslayoutitemattributetable.cpp

```diff
@@ -693,6 +706,66 @@ QVariant QgsLayoutItemAttributeTable::replaceWrapChar( const QVariant &variant )
   return replaced;
 }
 
+#ifdef HAVE_SERVER_PYTHON_PLUGINS
+QgsLayoutTableColumns QgsLayoutItemAttributeTable::filteredColumns()
+{
+  QgsLayoutTableColumns allowedColumns { mColumns };
```

Comment:
@elpaso can you add an early exit here if mLayout->renderContext().featureFilterProvider()  is null? that would limit the potential for regressions outside of the server...

Reply:
@nyalldawson sure, can we merge the backports and the I do an immediate followup? I think it's easier than patching the backport.

Reply:
Fair enough 😂